packages/integrations/netlify/README.md

```diff
@@ -92,7 +92,7 @@ export default defineConfig({
 
 ### Static sites
 
-For static sites you usually don't need an adapter. However, if you use `redirects` configuration (experimental) in your Astro config, the Netlify adapter can be used to translate this to the proper `_redirects` format.
+For static sites you usually don't need an adapter. However, if you use `redirects` configuration in your Astro config, the Netlify adapter can be used to translate this to the proper `_redirects` format.
```

Comment:
Nice catch! :raised_hands: 

Just confirming that there isn't something similar for Vercel 's README etc. that should also be removed.

Reply:
There's not